tests/modules/nf-core/sambamba/sort/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SAMBAMBA_SORT } from '../../../../../modules/nf-core/sambamba/sort/main.nf'
+
+workflow test_sambamba_sort {
```

Comment:
Can you add a test for CRAM files?

Reply:
sambamba sort only work with bam file (as far as I could understood from the documentation).